docs/detections/api/rules/rules-api-create-rule-default-exception-list.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[exceptions-api-create-rule-default-exception-list]]
+=== Create default exception list for a rule
+
+Creates a default exception list for the rule you specify. Default exception lists can only be associated with a single rule.
```

Comment:
I think the second sentence of this first paragraph would make more sense in the middle of the second paragraph, and the first sentence of the first paragraph would make more sense without it adjacent, too.

As an alternative, you also making the NOTE in this section a bit bigger by including this sentence as an additional point

Reply:
Yeah, I can see how these sentences repeat the same message. I like your suggestion to add the second sentence to the note. It'd be a good reminder why default exception lists can't be accessed from the Shared Exception Lists page.